x/leverage/fixtures/params.go

```diff
@@ -0,0 +1,18 @@
+package fixtures
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/umee-network/umee/v3/x/leverage/types"
+)
+
+// Params returns leverage params used in testing.
+func Params() types.Params {
+	return types.Params{
+		CompleteLiquidationThreshold: sdk.MustNewDecFromStr("0.1"),
```

Comment:
If https://github.com/umee-network/umee/pull/1409/ will get merged then we need to add the `_Trigger` param here.